### PR TITLE
Iek changes

### PR DESCRIPTION
**Pull request by charvolant**

### CODE REVIEW

**Comment by charvolant** [comment]
Includes fixes from code review of https://github.com/AtlasOfLivingAustralia/bie-plugin/pull/168

File: grails-app/taglib/au/org/ala/bie/BieTagLib.groovy [hunk 1]
```diff
@@ -185,31 +208,64 @@ class BieTagLib {
...
+        Locale lang = buildLocale(attrs.lang)
+        boolean format = attrs.containsKey('format') ? attrs.format : true
+        def name = languages[lang.language]
+        def display = name?.name ?: attrs.default ?: lang.displayName
+        if (!format) {
+            out << display
+        } else {
+            out << "<span class=\"language-name\" title=\"${name?.code ?: lang.language}\">"
+            if (name.uri)
+                out << "<a href=\"${name.uri}\" target=\"_blank\" onclick=\"window.open(this.href); return false;\">${display}</a>"
```
Comment (temi) [inline grails-app/taglib/au/org/ala/bie/BieTagLib.groovy]: onclick attribute can be removed.

File: grails-app/views/species/_names.gsp [hunk 2]
```diff
@@ -146,7 +146,80 @@
...
+                        <ul>
+                            <g:if test="${grailsApplication.config.vernacularName.pull.showLanguage}">
+                                <li>
+                                    <bie:showLanguage lang="${language}"/>
+                                </li>
+                            </g:if>
+                            <li>
+                                <g:if test="${cn.datasetURL}"><a href="${cn.datasetURL}"
+                                                                 onclick="window.open(this.href);
+                                                                 return false;">${cn.infoSourceName}</a></g:if>
```
Comment (temi) [inline grails-app/views/species/_names.gsp]: onclick attribute can be removed if you add target attribute here.

Comment (charvolant) [reply grails-app/views/species/_names.gsp]: Removed and added

File: grails-app/taglib/au/org/ala/bie/BieTagLib.groovy (hunk 1 above)
Comment (charvolant) [reply grails-app/taglib/au/org/ala/bie/BieTagLib.groovy]: Removed

**Pull request merged**